backend/app/controllers/company_updates_controller.rb

```diff
@@ -17,6 +17,10 @@ def request_update
     email = format_email(email)
 
     render json: { message: 'ok', email: email }, status: :ok
+    puts "TOKEN: "
+    puts token
+    #ApplicationMailer.company_update_token(email, token).deliver_now
```

Comment:
Parece código removido só para a finalidade de testes manuais. Se for este o caso, lembrem de remover o `#`